usersession/userd/launcher.go

```diff
@@ -80,6 +80,10 @@ var (
 		//   - scheme: apt:<name of package>
 		//   - https://github.com/snapcore/snapd/pull/7731
 		"apt",
+		// com.cloudflare.warp: the scheme allows for specifiying a package for
```

Comment:
The static checks are failing because of this typo
```suggestion
		// com.cloudflare.warp: the scheme allows for specifying a package for
```

Reply:
Whoops, thank you! That's what you get for last minute comment changes.